src/controllers/v1/entityTypes.js

```diff
@@ -241,7 +241,7 @@ module.exports = class EntityTypes extends Abstract {
 				const newEntityTypeData = await entityTypesHelper.bulkCreate(entityTypesCSVData, req.userDetails)
 
 				// Check if entity types were created successfully
-				if (newEntityTypeData.length > 0) {
+				if (newEntityTypeData.length > 0 && newEntityTypeData[0].status === CONSTANTS.apiResponses.SUCCESS) {
```

Comment:
@MallanagoudaB how are we notifying user if some entity creation failed ?

Reply:
This not what we discussed @VISHNUDAS-tunerlabs 
Its another ticket